packages/cli-lib/sandboxes.js

```diff
@@ -40,7 +51,65 @@ async function deleteSandbox(parentAccountId, sandboxAccountId) {
   };
 }
 
+/**
+ * Initiate a sync to a Sandbox portal.
+ * @param {Number} fromHubId - Source account for the sync
+ * @param {Number} toHubId - Target account for the sync
+ * @param {Array} tasks - Array of objects containing a type {Name} and portableKeys {Array}
+ * @param {Number} sandboxHubId - Sandbox portal ID for the sync
+ * @returns {Object} A new Sandbox portal instance.
+ */
+async function initiateSync(fromHubId, toHubId, tasks, sandboxHubId) {
+  let resp;
+
+  try {
+    resp = await _initiateSync(fromHubId, toHubId, tasks, sandboxHubId);
+  } catch (err) {
+    throw err;
+  }
+
+  return resp;
+}
+
+/**
+ * Fetches a task.
+ * @param {String} accountId - Parent account ID.
+ * @param {String} taskId - Sync task ID.
+ * @returns {Object} a sync task instance.
+ */
+async function fetchTaskStatus(accountId, taskId) {
+  let resp;
+
+  try {
+    resp = await _fetchTaskStatus(accountId, taskId);
+  } catch (err) {
+    throw err;
+  }
+
+  return resp;
+}
+
+/**
+ * Fetches available sync types for a specified portal.
```

Comment:
Would the types here be a regular sandbox vs a development sandbox?

Reply:
Commented about this here: https://github.com/HubSpot/hubspot-cli/pull/810#discussion_r1119101532